src/components/Header/ChainConnectivityWarning.tsx

```diff
@@ -2,17 +2,25 @@ import { Trans } from '@lingui/macro'
 import { useWeb3React } from '@web3-react/core'
 import { getChainInfoOrDefault, L2ChainInfo } from 'constants/chainInfo'
 import { SupportedChainId } from 'constants/chains'
-import { AlertOctagon } from 'react-feather'
+import { RedesignVariant, useRedesignFlag } from 'featureFlags/flags/redesign'
+import { AlertOctagon, AlertTriangle } from 'react-feather'
 import styled from 'styled-components/macro'
 import { ExternalLink, MEDIA_WIDTHS } from 'theme'
 
-const BodyRow = styled.div`
-  color: ${({ theme }) => theme.deprecated_black};
+const BodyRow = styled.div<{ redesignFlag?: boolean }>`
```

Comment:
i found that there's a dom warning since these props get passed to the dom. i don't think it's necessary to redo this, but in the future using a $ will prevent that: https://styled-components.com/docs/api#transient-props

Reply:
done thanks!!